python/tk_multi_reviewsubmission/renderer.py

```diff
@@ -1,187 +1,210 @@
 # Copyright (c) 2013 Shotgun Software Inc.
-# 
+#
 # CONFIDENTIAL AND PROPRIETARY
-# 
-# This work is provided "AS IS" and subject to the Shotgun Pipeline Toolkit 
+#
+# This work is provided "AS IS" and subject to the Shotgun Pipeline Toolkit
 # Source Code License included in this distribution package. See LICENSE.
-# By accessing, using, copying or modifying this work you indicate your 
-# agreement to the Shotgun Pipeline Toolkit Source Code License. All rights 
+# By accessing, using, copying or modifying this work you indicate your
+# agreement to the Shotgun Pipeline Toolkit Source Code License. All rights
 # not expressly granted therein are reserved by Shotgun Software Inc.
 
 import sgtk
 import os
 import sys
-import nuke
+
+import subprocess
+
+try:
+    import nuke
+except:
+    nuke = None
+
 
 class Renderer(object):
-    
     def __init__(self):
         """
         Construction
         """
-        self.__app = sgtk.platform.current_bundle() 
-        
-        self._burnin_nk = os.path.join(self.__app.disk_location, "resources", "burnin.nk")
+        self.__app = sgtk.platform.current_bundle()
         self._font = os.path.join(self.__app.disk_location, "resources", "liberationsans_regular.ttf")
-        
-        # If the slate_logo supplied was an empty string, the result of getting 
-        # the setting will be the config folder which is invalid so catch that
-        # and make our logo path an empty string which Nuke won't have issues with.
+
+        burnin_template = self.__app.get_template("burnin_path")
+        self._burnin_nk = burnin_template.apply_fields({})
```

Comment:
I know for our particular use-case, since the burnin file is stored at the project level, this will resolve fine, but we should make sure we're passing context fields to this method in case in the future we choose to localize this file to a seq/shot.

Reply:
Good point. Passing on context.as_template_fields().